types/openlayers/openlayers-tests.ts

```diff
@@ -608,10 +609,10 @@ writeOptions.featureProjection = projection;
 writeOptions.rightHanded = false;
 
 // Test functions
-feature = geojsonFormat.readFeature("json");
-feature = geojsonFormat.readFeature("json", readOptions);
-featureArray = geojsonFormat.readFeatures("json");
-featureArray = geojsonFormat.readFeatures("json", readOptions);
+featureFormat = geojsonFormat.readFeature("json");
```

Comment:
These changes are not ok, because readFeature should return an ol.Feature, not an ol.format.Feature